src/config.ts

```diff
@@ -75,4 +75,4 @@ export const libDirPath = path.resolve(baseDirPath, libDirName);
 /** Absolute path to the language and diagram server jar. */
 // TODO handle version in file name
 export const sourceLdsJarFile = path.resolve(baseDirPath,
-    path.join(repoName, 'org.lflang.diagram', 'build', 'libs', 'org.lflang.diagram-0.3.1-SNAPSHOT-lds.jar'));
+    path.join(repoName, 'org.lflang.diagram', 'build', 'libs', 'org.lflang.diagram-+(?).+(?).+(?)?(-SNAPSHOT)-lds.jar'));
```

Comment:
```suggestion
    path.join(repoName, 'org.lflang.diagram', 'build', 'libs', 'org.lflang.diagram-+(?).+(?).+(?)?(-SNAPSHOT)?-lds.jar'));
```

Reply:
There is a weird syntax for `?` wildcards, according to the [`glob` documentation](https://www.npmjs.com/package/glob):
>  `?(pattern|pattern|pattern)` Matches zero or one occurrence of the patterns provided.

Reply:
Oh, you're right. Weird syntax indeed. OK, you can strike my suggestion then. Looks all good.